docs/upgrading.md

```diff
@@ -131,7 +131,7 @@ There are 4 main differences on the new API:
 
 1. Token configuration methods were renamed
 1. Signature is created via `Builder#getToken()` (instead of `Builder#sign()`)
-1. `DateTimeImmutable` objects are now for the registered claims with dates
+1. `DateTimeImmutable` objects are now used for the registered claims with dates and will use microtime instead of unix timestamps
```

Comment:
I like the explicitness here, just think that we can improve the construction. What do you think about this:

```suggestion
1. `DateTimeImmutable` objects are now used for the registered claims with dates, which are by default encoded as floats with microseconds as precision
```